main.tf

```
@@ -222,8 +222,8 @@ resource "aws_security_group_rule" "allow_icmp_ingress" {
   security_group_id = join("", aws_security_group.ecs_service.*.id)
 }
 
-resource "aws_security_group_rule" "alb" {
-  count                    = var.enabled ? 1 : 0
+resource "aws_security_group_rule" "default" {
```

```suggestion
resource "aws_security_group_rule" "alb" {
```

it's a rule for ALB, let's keep the name